todo_app/data/trello_items.py

```diff
@@ -18,7 +18,7 @@ def get_trello_items():
     secrets = get_secrets()
 
     url = f"https://api.trello.com/1/boards/{secrets['board_id']}/lists/"
-
+    print(url)
```

Comment:
Given that we're claiming that `board_id` is a secret, we shouldn't really end up printing it to our logs! I know that it doesn't really have to be treated as a secret, but I think this is probably still worth removing after you've finished using it for debugging since it won't add much value without context